src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -81,6 +81,9 @@ extends AbstractMap[Key, Value]
     h ^ (h >>> 7) ^ (h >>> 4)
   }
 
+  /** Increase the size of the table.
+    * Copy only the occupied slots, effectively eliminating the deleted slots.
+    */
```

Comment:
As long as you are making documentation changes, can you make the return type on `growTable()` explicit?